StatBot/Models/Statistic.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+
+namespace StatBot
+{
+    public class Statistic
+    {
+        public Statistic(string name, string message, string query)
+        {
+            Name = name;
+            Message = message;
+            Query = query;
+            Subscribers = new List<string>();
+            IsActive = true;
+        }
+
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Message { get; set; } // выдаваемое сообщение
+        public string Query { get; set; } // sql-запрос
+        public List<string> Subscribers { get; set; }
+        public bool IsActive { get; set; } = true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the member initializer, all constructors set an initial value for the member. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS3604)